index.html

```diff
@@ -4035,7 +4023,7 @@ <h2>
 and fragment. This process depends on <a>DID resolution</a> of the <a>DID</a>
 contained in the <a>DID URL</a>.
 The details of how this process is accomplished are outside the scope of this
-specification, but all conformant implementations MUST implement a function
+specification, but all conformant implementations implement a function
```

Comment:
How would we write a test to ensure that a conformant implementation implements a function with the following *abstract* form. That is, how do we test to make sure an abstract function is defined... since by it's very nature, the function is abstract and has no testable realization?